### ISSUE TITLE
Modernize our build process

### ISSUE THREAD

**Issue by cjyabraham** [cncf/cncf.io#597]
Our build process is about 3.5 years old, is not supported by anyone, has some [new issues with it](https://github.com/cncf/cncf.io/issues/593), and doesn't use Pantheon's new build tools. Start from scratch using [Pantheon's build tools](https://pantheon.io/docs/guides/build-tools/) ([repo is here](https://github.com/pantheon-systems/terminus-build-tools-plugin)) to get on a build system that is supported and maintained.

It looks like these build tools support [a GitHub Actions option](https://github.com/pantheon-systems/terminus-build-tools-plugin#available-services) which we could possible use instead of CircleCI to keep all code and build on one platform.

**Comment by thetwopct** [comment]
Perhaps this could be done alongside with making sure project onboarding / developer instructions is up to date and still relevant - guess that will need to be done before new team joins us on November 1st.

**Comment by cjyabraham** [comment]
I looked into this and learned a few things:
- The new [Pantheon build tools](https://pantheon.io/docs/guides/build-tools/) could be useful for us next time we create a new WP site. They set up new WP sites with a repo, CI scaffolding, and Pantheon instance, so that's the only reason we'd use them.
- The testing scaffolding that the build tools creates provide behat and visual regression testing but we don't have an immediate need for these on our current sites
- The build tools provide the option of using GitHub Actions instead of CircleCI. I set up a test instance with GitHub Actions configured and it seemed to work fine. This might be nice to try for our next WP site as it would save time and wouldn't require separate accounts etc. with CircleCI. There's no reason to bother moving our current sites to GitHub Actions since our current CircleCI processes are fine.
- One lingering issue I sometimes see with the LFEvents build is a "wasm out-of-memory" error but this occurs during the css/js build process so could be addressed independently to this issue.

In summary, I don't think there's any immediate need to change our existing sites and configurations. Next time we need to start a new WP instance, though, we could consider using the new build tools if there's some advantage to doing that and deviating from our current pattern.

As for the new people getting local instances of the CNCF site set up, it'll be interesting to see if they run into issues. Having just tried them, however, I think [the instructions](https://github.com/cncf/cncf.io) should work as they are.
